hathor/prometheus.py

```diff
@@ -26,6 +26,8 @@
 
 settings = HathorSettings()
 
+METRIC_PREFIX = 'hathor_core:'
```

Comment:
We are not using this anymore, right?

Reply:
Sure, I must have overseen this during merge